disgames/mixins/hangman.py

```diff
@@ -8,6 +9,35 @@ class Hangman(commands.Cog):
 
     def __init__(self, bot):
         self.bot = bot
+        
+    @property
+    def _url(self):
+        return "https://raw.githubusercontent.com/andrewthederp/Disgames/main/disgames/mixins/words.txt"
+    
+    @property
+    def _session(self):
+        return self.bot.http._HTTPClient__session
+    
+    async def _request(self):
+        response = await self._session.get(self._url)
+        return await response.text()
+    
+    async def _get_word(self):
+        try:
+            with open('./words.txt', 'r') as file:
+                data = file.read().splitlines()
+                word = random.choice(data)                
+        except as Exception:
+            words = await self.request()
```

Comment:
im pretty sure this should be `self._request`

Reply:
Will fix